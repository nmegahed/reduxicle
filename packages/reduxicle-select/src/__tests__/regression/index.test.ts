import * as api from "../../index";

describe("index", () => {
  it("should not have changed API", () => {
    expect(api).toMatchSnapshot();
  });
});
