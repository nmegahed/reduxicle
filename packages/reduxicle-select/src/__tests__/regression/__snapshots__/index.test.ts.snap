// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should not have changed API 1`] = `
Object {
  "withSelectItem": [Function],
}
`;
